### PR TITLE
Code preview

### PR DESCRIPTION
**Pull request by amicis31**
# Update in the code preview or template feature #89 :
- The code template can be modified or updated on the 'task edit' section

- Various languages can be added at once

### CODE REVIEW

**Review by crdgonzalezca — CHANGES_REQUESTED** [review]
Nice, some few comments. Also, remove code_preview.js file as it is empty.

File: inginious/frontend/plugins/code_preview/__init__.py [hunk 1]
```diff
@@ -10,6 +11,8 @@ def init(plugin_manager, course_factory, client, config):
     plugin_manager.add_page('/api/code_preview/', TaskPreviewFileAPI)
     plugin_manager.add_page(r'/code_preview/static/(.*)', create_static_resource_page(_static_folder_path))
 
+    plugin_manager.add_hook('task_editor_tab', code_preview_tab)
+    plugin_manager.add_hook('task_editor_submit', on_task_editor_submit)    
+    plugin_manager.add_hook("javascript_footer", lambda: "/code_preview/static/js/code_preview_load.js")
```
[the remark this answers is not in the document]
Comment (amicis31) [reply inginious/frontend/plugins/code_preview/__init__.py]: Done

File: inginious/frontend/plugins/code_preview/pages/code_preview.html [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+          }
+
+          let entry = $$("<option>", {
+            "value": file.complete_name,
+            "text": file.complete_name
+          });
+
+          previewFileSelect.append(entry);
+
+        });
```
[the remark this answers is not in the document]
Comment (amicis31) [reply inginious/frontend/plugins/code_preview/pages/code_preview.html]: Done

**Review by crdgonzalezca — APPROVED** [review]
LGTM.

Did you update the minifier program algo with this plugin?

**Comment by amicis31** [comment]
Yes, I will uploaded now

**Pull request merged**